ethcore/src/trace/types/error.rs

```diff
@@ -57,7 +63,9 @@ impl<'a> From<&'a VmError> for Error {
 			VmError::BadInstruction { .. } => Error::BadInstruction,
 			VmError::StackUnderflow { .. } => Error::StackUnderflow,
 			VmError::OutOfStack { .. } => Error::OutOfStack,
-			VmError::BuiltIn { .. } => Error::BuiltIn,
+			VmError::SubStackUnderflow { .. } => Error::SubStackUnderflow,
+			VmError::OutOfSubStack { .. } => Error::OutOfSubStack,
+			VmError::InvalidSubEntry { .. } => Error::InvalidSubEntry,			VmError::BuiltIn { .. } => Error::BuiltIn,
```

Comment:
one more here, sorry i miss it first time.

Reply:
```suggestion
			VmError::InvalidSubEntry { .. } => Error::InvalidSubEntry,
			VmError::BuiltIn { .. } => Error::BuiltIn,
```